otherlibs/dune-site/src/plugins/linker/toplevel/linker.ml

```diff
@@ -11,6 +11,8 @@ open Toploop [@@ocaml.warning "-33"]
 let load filename =
   let buf = Buffer.create 16 in
   let ppf = Format.formatter_of_buffer buf in
+  if Dune_site_private.debug then
+    Printf.printf "DUNE_SITE_DEBUG[toplevel]: loading %s\n" filename;
```

Comment:
Can you add a test that uses that code path?

Reply:
If the original issue has been fixed with just the other print, I'd say we leave that part out of the first version.

Reply:
There is no issue that we are actually fixing here. We are simply giving the user of the library the option to output some extra debugging information. I don't see why it would be harmful to print stuff even if we aren't testing it here?

The use case is extremely niche and is really a quality-of-life fix.